tests/src/test/scala/org/http4s/StaticFileSpec.scala

```diff
@@ -276,5 +276,14 @@ class StaticFileSpec extends Http4sSpec with Http4sLegacyMatchersIO {
           .map(_.flatMap(_.contentLength))
       len must returnValue(Some(24005L))
     }
+
+    "Read a directory from a URL" in {
```

Comment:
Maybe say "return none from a URL that is a directory".  I was confused what "reading a directory" meant until I reread the linked issue.

Reply:
resolved